pubsub/callback/callback.go

```diff
@@ -0,0 +1,41 @@
+package callback
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/micromdm/micromdm/pubsub"
+)
+
+type CallbackPublisher struct {
+	url  string
+	next pubsub.Publisher
+}
+
+func (p *CallbackPublisher) Publish(ctx context.Context, topic string, msg []byte) error {
+
+	// send to callback url
+	var buf bytes.Buffer
+	var event = struct {
+		Topic   string `json:"topic"`
+		Message []byte `json:"message"`
+	}{
+		Topic:   topic,
+		Message: msg,
```

Comment:
_Otherwise_ yeah, this is fine for e.g. all the other events. There is no command response event so we still end up needing to code a new event to publish on the bus as a special case.